programmers/난이도별/level01/수박수박수박수/HyeonJeong.py

```diff
@@ -0,0 +1,20 @@
+# 방법 1
+'''
+def solution(n):
+    answer = ''
+    word1 = '수'
+    word2 = '박'
+    for i in range(n) :
+        if i % 2 == 0 :
+            answer += word1
+        else :
+            answer += word2
+    return answer
+'''
+
+# 방법 2(더 간단하고 효율적인 코드로)
+def solution(n):
+    answer = '수박' * (n//2) + '수' * (n % 2)
+    return answer
```

Comment:
```suggestion
    return '수박' * (n//2) + '수' * (n % 2)
```

바로 리턴하면 됩니다!
잘 풀이하셨어요~ 💯